osu.Game/Overlays/Dashboard/CurrentlyPlayingDisplay.cs

```diff
@@ -138,7 +139,7 @@ private void load(IAPIProvider api)
                             new PurpleTriangleButton
                             {
                                 RelativeSizeAxes = Axes.X,
-                                Text = "Watch",
+                                Text = CommonStrings.ButtonsWatchTo1,
```

Comment:
Not sure about this one. The contextual nature of "watch" is probably different here.

The common translation here is to "watch" or "unwatch" an object, whereas the usage here is to watch/spectator a player.

I think we should make this its own translation for now to avoid confusion in the future.